src/Data/Morpheus/Server/ClientRegister.hs

```diff
@@ -64,7 +64,7 @@ updateClientByID id' updateFunc state = modifyMVar_ state (return . map updateCl
 publishUpdates :: (Eq s) => GQLState IO s -> PubPair s -> IO ()
 publishUpdates state (channels, value) = do
   state' <- readMVar state
-  forM_ state' sendMessage
+  traverse_ sendMessage state'
```

Comment:
Unrelated to this PR. `forM_` is now `for_` which is just a flipped version of `traverse_` and I think the latter is more standard so I took the liberty of changing this but let me know if you don't like this, I can remove it.